src/services/auth.service.ts

```diff
@@ -0,0 +1,20 @@
+import axios from "axios";
+
+const API = "http://localhost:3500";
+
+export const signUp = async (credentials: object) => {
```

Comment:
(suggestion; non-blocking): The `object` type is good, but we should consider creating an interface that better describes what object properties this API endpoint accepts. For example:

```
interface SignUpParams {
  email: string;
  password: string;
}

export const signUp = async (credentials: SignUpParams) => { ... }
```